R/teal_data_module.R

```diff
@@ -48,9 +50,8 @@
 #' )
 #'
 #' @name teal_data_module
-#' @rdname teal_data_module
 #'
-#' @seealso [`teal_data-class`], [`base::within()`], [`teal.code::within.qenv()`]
+#' @seealso [teal.data::`teal_data-class`], [teal.code::qenv()]
```

Comment:
![image](https://github.com/insightsengineering/teal/assets/211358/5c000c27-1849-47e1-82de-861fb88442d9)

```suggestion
#' @seealso [`teal.data::teal_data-class`], [teal.code::qenv()]
```

Reply:
That format has a small issue as the formatting is different from `[fun()]`. It's small, but visible

I'd still use ``[`teal.data::teal_data-class`]`` for consistency in the `@seealso` tag

![image](https://github.com/insightsengineering/teal/assets/211358/7047e275-8cae-4db8-bc4f-2be5aa141fd6)

Reply:
That glitch is common to all special character function names :-)

Reply:
> I'd still use [`teal.data::teal_data-class`] for consistency

Yes updated this already . [bb904f0](https://github.com/insightsengineering/teal/pull/1021/commits/bb904f0704ccf8ffec1854a33a47ee45468d4e55)